api-spec.md

```diff
@@ -30,20 +30,20 @@ Login
 Reporting
 ---------
 
-#### Report Diagnosis: `POST /reports/`
+#### Report Diagnosis: `POST /cenreport/`
 #### Auth Required: Yes
 #### Body Example:
 ```
 {
-   "device_id": "xxxxxx"
-   "ha_token": "xxxxxx",
-   “info_questions”: {
-        “question_1”: “answer_1”,
-        “question_2”: “answer_2”,
-        ...
-     },
-   “status”: "confirmed",
-   “reported_at”: “timestamp”
+  "validationPin": "string_token",
+  "traces": [
+    { geo_hash }
+  ],
+  "report": "fever, cough, headach",
+  "cenKeys": [
+    "key_1",
+    "key_2",
+  ]
```

Comment:
I am not sure if we'll support uploading symptoms (at least in the first iteration). I think for now we expect people to upload report when they have been tested.

 Also, if we go with latest version of TCN protocol described in https://github.com/TCNCoalition/TCN, we should expect TCN report to be a binary data serialized as hex-encoded string. 

For future compatibility we can still have an optional field "symptoms" if the reported doesn't submit a confirmed case.
So to me the body would look like:
```suggestion
  "validationPin": "string_token",
  "symptoms": "fever, cough, headache",
  "traces": [ "geo_hash1", "geo_hash2" ],
  "tcnData": "c432bf9538f248a8238543a89d08e098c89bb890"
```

Even if their current implementation doesn't support this kind of submission yet, their protocol description implies the submitted data to be something of this kind.

Reply:
Updated suggestion after removing symptoms part:

```suggestion
  "validationPin": "string_token",
  "traces": [ "geo_hash1", "geo_hash2" ],
  "tcnData": "c432bf9538f248a8238543a89d08e098c89bb890"
```

Reply:
ohh okay, so client side will encode the report and all the server is storing is this encoded string

Reply:
Yes, exactly. We may also add some "decoding" on server side in the future (e.g. in case we include symptom information in TCN data for some analytics), but for simplicity it should be just data saved as is.